app/models/restaurant.rb

```diff
@@ -0,0 +1,38 @@
+# == Schema Information
+#
+# Table name: restaurants
+#
+#  id                    :uuid             not null, primary key
+#  name                  :string           not null
+#  rating                :integer          default(0)
+#  accept10bis           :boolean          default(FALSE), not null
+#  max_delivery_time_min :integer
+#  address               :string
+#  latitude              :float
+#  longitude             :float
+#  cuisine               :string
+#  created_at            :datetime         not null
+#  updated_at            :datetime         not null
+#
+
+class Restaurant < ApplicationRecord
+  validates :rating, numericality: { greater_than_or_equal_to: 0,
+                                     less_than_or_equal_to: 3 }
+  validates :name, presence: true
```

Comment:
Can you use more of `validate_presence_of` and its friends?

Reply:
[https://www.linkedin.com/learning/ruby-on-rails-5-essential-training/validates-method?u=2160057](url)
these **validates-methods** are considered clearer then the explicit multiple lines